packages/@vue/cli-shared-utils/lib/module.js

```diff
@@ -35,9 +51,13 @@ const resolve = semver.satisfies(process.version, '>=10.0.0')
 exports.resolveModule = function (request, context) {
   let resolvedPath
   try {
-    resolvedPath = resolve(request, {
-      paths: [context]
-    })
+    try {
+      resolvedPath = createRequire(context).resolve(request)
```

Comment:
Is `context` a file, or a directory? I think `createRequire` only takes files, so worst case you can `join` it to a fake filename.

Reply:
👍It's a directory. I've got it fixed now